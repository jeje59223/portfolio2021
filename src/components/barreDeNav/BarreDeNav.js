// eslint-disable-next-line no-unused-vars
import React from 'react';
import moi from '../../images/moi.jpeg';
// eslint-disable-next-line no-unused-vars
import TextIntro from '../TextIntro';
import '../../styles/barreDeNav.css';

const BarreDeNav = () => (
        <header id="top">
            <div className="container">
                <nav className="navbar navbar-expand-md bg-dark navbar-dark fixed-top" id="main-nav">
                    <div className="container">
                        <a href="index.html" className="navbar-brand"><img src={moi} alt="moi"/></a>
                        <button className="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
                            <span className="navbar-toggler-icon"></span>
                        </button>
                        <div className="collapse navbar-collapse" id="navbarCollapse">
                            <ul className="navbar-nav ml-auto">
                                <li className="nav-item mr-3">
                                    <a href="#top" className="nav-link">Accueil</a>
                                </li>
                                <li className="nav-item mr-3">
                                    <a href="#presentation" className="nav-link">Présentation</a>
                                </li>
                                <li className="nav-item mr-3">
                                    <a href="#experiencesProfessionnelles" className="nav-link">Expériences Professionnelles</a>
                                </li>
                                <li className="nav-item mr-3">
                                    <a href="#competences" className="nav-link">Compétences</a>
                                </li>
                                <li className="nav-item mr-3">
                                    <a href="#portfolio" className="nav-link">Projets</a>
                                </li>
                                <li className="nav-item mr-3">
                                    <a href="#contact" className="nav-link">Contact</a>
                                </li>
                            </ul>
                        </div>
                    </div>
                </nav>
                <TextIntro />
            </div>
        </header>
);

export default BarreDeNav;
