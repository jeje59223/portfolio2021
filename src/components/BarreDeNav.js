import React from "react";
import moi from "../images/moi.jpeg";
import TextIntro from "./TextIntro";
import "../styles/barreDeNav.css";

const BarreDeNav = () => {
    return (
        <header id="top">
            <div class="container">
                <nav class="navbar navbar-expand-md bg-dark navbar-dark fixed-top" id="main-nav">
                    <div class="container">
                        <a href="index.html" class="navbar-brand"><img src={moi} alt="moi"/></a>
                        <button class="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
                            <span class="navbar-toggler-icon"></span>
                        </button>
                        <div class="collapse navbar-collapse" id="navbarCollapse">
                            <ul class="navbar-nav ml-auto">
                                <li class="nav-item mr-3">
                                    <a href="#top" class="nav-link">Accueil</a>
                                </li>
                                <li class="nav-item mr-3">
                                    <a href="#presentation" class="nav-link">Présentation</a>
                                </li>
                                <li class="nav-item mr-3">
                                    <a href="#experiencesProfessionnelles" class="nav-link">Expériences Professionnelles</a>
                                </li>
                                <li class="nav-item mr-3">
                                    <a href="#competences" class="nav-link">Compétences</a>
                                </li>
                                <li class="nav-item mr-3">
                                    <a href="#portfolio" class="nav-link">Projets</a>
                                </li>
                                <li class="nav-item mr-3">
                                    <a href="#contact" class="nav-link">Contact</a>
                                </li>
                            </ul>
                        </div>
                    </div>
                </nav>
                <TextIntro />
            </div>
        </header>
    )
}

export default BarreDeNav;