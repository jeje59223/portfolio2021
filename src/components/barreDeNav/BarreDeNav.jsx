import React from 'react';
import moi from '../../images/moi.jpeg'
import '../../styles/barreDeNav.css';

const BarreDeNav = () => (
  <header id="top">
    <div className="container">
      <nav className="navbar navbar-expand-md bg-dark navbar-dark fixed-top" id="main-nav">
        <div className="container">
          <a href="#top" className="navbar-brand"><img src={moi} alt="moi" /></a>
          <button type="button" className="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
            <span className="navbar-toggler-icon" />
          </button>
          <div className="collapse navbar-collapse" id="navbarCollapse">
            <ul className="navbar-nav ml-auto">
              <li className="nav-item mr-3">
                <a href="#top" className="nav-link">Accueil</a>
              </li>
              <li className="nav-item mr-3">
                <a href="#presentation" className="nav-link">Présentation</a>
              </li>
              <li className="nav-item mr-3">
                <a href="#experiencesProfessionnelles" className="nav-link">Expériences Professionnelles</a>
              </li>
              <li className="nav-item mr-3">
                <a href="#competences" className="nav-link">Compétences</a>
              </li>
              <li className="nav-item mr-3">
                <a href="#portfolio" className="nav-link">Projets</a>
              </li>
              <li className="nav-item mr-3">
                <a href="#contact" className="nav-link">Contact</a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    </div>
  </header>
);

export default BarreDeNav;
