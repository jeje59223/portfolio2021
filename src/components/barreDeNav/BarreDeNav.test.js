import toJson from 'enzyme-to-json';
import React from 'react';
import BarreDeNav from "./BarreDeNav";
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('<BarreDeNav>', () => {
  it('should render Navigation Bar', () => {
    const wrapper = shallow(<BarreDeNav/>);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});