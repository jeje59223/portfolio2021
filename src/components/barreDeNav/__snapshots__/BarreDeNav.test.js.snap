// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BarreDeNav> should render Navigation Bar 1`] = `
<header
  id="top"
>
  <div
    className="container"
  >
    <nav
      className="navbar navbar-expand-md bg-dark navbar-dark fixed-top"
      id="main-nav"
    >
      <div
        className="container"
      >
        <a
          className="navbar-brand"
          href="index.html"
        >
          <img
            alt="moi"
            src="moi.jpeg"
          />
        </a>
        <button
          className="navbar-toggler"
          data-target="#navbarCollapse"
          data-toggle="collapse"
        >
          <span
            className="navbar-toggler-icon"
          />
        </button>
        <div
          className="collapse navbar-collapse"
          id="navbarCollapse"
        >
          <ul
            className="navbar-nav ml-auto"
          >
            <li
              className="nav-item mr-3"
            >
              <a
                className="nav-link"
                href="#top"
              >
                Accueil
              </a>
            </li>
            <li
              className="nav-item mr-3"
            >
              <a
                className="nav-link"
                href="#presentation"
              >
                Présentation
              </a>
            </li>
            <li
              className="nav-item mr-3"
            >
              <a
                className="nav-link"
                href="#experiencesProfessionnelles"
              >
                Expériences Professionnelles
              </a>
            </li>
            <li
              className="nav-item mr-3"
            >
              <a
                className="nav-link"
                href="#competences"
              >
                Compétences
              </a>
            </li>
            <li
              className="nav-item mr-3"
            >
              <a
                className="nav-link"
                href="#portfolio"
              >
                Projets
              </a>
            </li>
            <li
              className="nav-item mr-3"
            >
              <a
                className="nav-link"
                href="#contact"
              >
                Contact
              </a>
            </li>
          </ul>
        </div>
      </div>
    </nav>
    <TextIntro />
  </div>
</header>
`;
